### PR TITLE
Preserve data

### PR DESCRIPTION
**Pull request by AshwinCarvalho999**
## Learning objectives
- Stored the data in files.
- Run a program using the command line.
- On exit our app save all the data (books, people, and rentals) into JSON files. One per type of data you are storing and named accordingly.
- On startup of our app it load all data from files.

### CODE REVIEW

**Comment by prolajumokeoni** [comment]
Hi @AshwinCarvalho999 and @j-tee, Kindly add the key features and future features of your application to your readme 👍🏿

**Comment by prolajumokeoni** [comment]
Hi @Team

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that does not have the key features and future features  section in your readme

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: main.rb [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  puts
+  puts 'Please choose an option by entrying a number:'
+
+  select_option = [
+    '1 - List all people',
+    '2 - List all books',
+    '3 - Create a person',
+    '4 - Create a book',
+    '5 - Create a rental',
+    '6 - List all rentals for a given person id',
```
Comment (Ugboaja-Uchechi) [inline main.rb:15]: - [ ] Great job 👍. When creating a rental it gives this error. Kindly fix it 🙏 

image

**Review by Ugboaja-Uchechi — CHANGES_REQUESTED** [review]
Hi @AshwinCarvalho999 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

- Linters are passing ✔️ 
- README is descriptive 👍 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request closed without merging**